node_tree.py

```diff
@@ -346,6 +353,7 @@ def copy(self, original):
             self.use_custom_color, self.color = settings
         self.sv_copy(original)
         self.n_id = ""
+        self.sv_copy_post(original)
```

Comment:
```suggestion
        self.n_id = ""
        self.sv_copy(original)
```

Reply:
if we never need to know the current node_id before copying, then yes, perhaps `sv_copy_post` is not needed.

Reply:
ok , lets do that.